### PR TITLE
Pages updated to look better

### PR DESCRIPTION
**Pull request by reich6534**

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -2,7 +2,7 @@
 from bmc.models import User, Practice
 
 if (__name__ == '__main__'):
-    bmc_app.run(debug=False)
+    bmc_app.run(debug=True)
```
Comment (awangc) [inline app.py:5]: I think the `debug=True` should be removed. In order to debug, we can set the environment variable: in your powershell/cmd, do: `set FLASK_ENV=development`. See it [here](https://flask.palletsprojects.com/en/1.1.x/config/#environment-and-debug-features)

File: bmc/routes.py [hunk 2]
```diff
@@ -84,19 +85,19 @@ def login():
         login_user(user, remember=form.remember_me.data)
         next_page = request.args.get('next')
         if (not next_page or url_parse(next_page).netloc != ''):
-            next_page = url_for('potato')
+            next_page = url_for('enter')
```
Comment (awangc) [inline bmc/routes.py:88]: Giving up on `potato`? 😂

File: migrations/versions/474672feeb0b_first_init.py [hunk 3]
```diff
@@ -1,16 +1,16 @@
-"""initial migration
+"""first init
```
Comment (awangc) [inline migrations/versions/474672feeb0b_first_init.py:1]: I think these migration files should be removed. It does not seem to help. At least not right now.

**Comment by andrewkewang** [comment]
... and of course all the comments and approvals were from the wrong profile!

**Pull request merged**